docs/guides/debugging.md

```diff
@@ -149,6 +149,30 @@ places. You can always load it by hand if GDB refuses or fails to load it:
 
     (gdb) source /path/to/your/.gdbinit
 
+Here's an example of a `.gdbinit` file helpful for debugging scylla.
+Note that it registers pretty-printer helpers for printing common
+std-c++ stl containers:
+```
+handle SIG34 pass noprint
+handle SIGUSR1 pass noprint
+set print pretty
+set print static-members no
+set python print-stack full
+set auto-load safe-path /opt/scylladb/libreloc
+add-auto-load-safe-path /lib64
+add-auto-load-safe-path /usr/lib64
+set solib-search-path /opt/scylladb/libreloc:/lib64:/usr/lib64
+set pagination off
+set debug libthread-db 1
```

Comment:
This makes the initial loading of the core quite verbose but we can leave it in.